lesson03/assignment/css/_leftCard.scss

```diff
@@ -0,0 +1,33 @@
+// for picture on the left side
+
+.pic {
+	@include border-radius($radius);
+	/* border: 10px solid green;
+		overflow: auto;  */
+	 width: 230px;
+	 height: 200px;
```

Comment:
The height here is causing issues in mobile, you'd be better suited to leave container height set to auto and control the elements within to dictate the height via content.